File: minwook/3주차/짝수_홀수_개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2]; // 짝수, 홀수 배열 공간
        for(int n : num_list){ // 배열 값 전부 접근
            if(n%2 == 0) // 짝수일 경우
                answer[0]++;
            else // 홀수일 경우
                answer[1]++;
        }
        return answer;
    }
}

File: yongkeun/3week/짝수홀수개수.js
function solution(num_list) {
  let even = 0; //짝수
  let odd = 0; //홀수
  num_list.forEach((value) => {
    if (value % 2 == 0) {
      //짝수면
      even++; //짝수더하기
    } else if (value % 2 == 1) {
      odd++;
    }
  });
  return [even, odd];
}
